### DISCUSSION TITLE
Adding monsters

### DISCUSSION THREAD

**Discussion by Elemperor1** [Tuxemon/Tuxemon#2309] [General]
Hi there,
I would like to add monsters/maps/story to the game, where would I put these?

**Comment by Sanglorian** [comment]
That would be great, thank you! 

You might want to post them here first for discussion. Then people will be able to point you in the right direction.

**Comment by HippasusTwo** [comment]
I made up some evos and sprites based on existing art for Fruitera.
You can see where I added the sprites to what folders here:
51be407aaecfea320e27b3e094e34016e2dea15d
To have it fully functional in the game, you'd also need to alter the db and give it some moves. To add it to Tuxepedia and have it 'officially' part of the game, you'd have to let other people know here what you did.
